lib/Worker.js

```diff
@@ -110,7 +110,7 @@ module.exports = (function(){
         });
         _.forEach(alreadyLiveEntries, function(e) {
             var entry = that.handledEntries[e.entryId];
-            if (entry !== 'pending')
+            if (entry !== 'pending' && entry !== 'cancelPending')
```

Comment:
@gadyaari  Gad, can you please talk with Igor about the cancelPending to see if this is missing from 11.0 as well